app/controllers/collections_controller.rb

```diff
@@ -95,14 +97,15 @@ def view_shared
   end
 
   def save_shared
-    return redirect_to user_messages_path(current_user), alert: t('.errors.already_member') if @collection.users.include? current_user
+    user = current_user
```

Comment:
Why is the `current_user` assigned here? 🤔

Reply:
We need `current_user` 5 times in this method and with that the Metrics/AbcSize-Cop triggers. An assignment beforehand alleviates that issue.